### PR TITLE
95% payroll-frame

### PR DESCRIPTION
**Pull request by CloudyMushroom**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

# Type of change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

# Screenshots (if appropriate)

Add screenshots that could help the reviewer.

# Checklist:

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by CloudyMushroom** [comment]
Itong part baka merong library yung react para sa scrollable year/date picker? so depende na lang sa inyo. Once na integrate na yung scrollable year/date.(kung meron man) then if may papadagdag na design let me know na lang.

**Comment by franzegos** [comment]
nice, @glendell03  okay lang palipat ulit to pri? yung nasa auditor.attendance ililipat sana ditoo 
image

**Comment by glendell03** [comment]
> 
> image
> image
> Itong part baka merong library yung react para sa scrollable year/date picker? so depende na lang sa inyo. Once na integrate na yung scrollable year/date.(kung meron man) then if may papadagdag na design let me know na lang.

di ko lang sure pero pwede ata gamitin yung Date ng Js dyan

**Comment by glendell03** [comment]
> nice, @glendell03  okay lang palipat ulit to pri? yung nasa auditor.attendance ililipat sana ditoo 
> image
> 

sige sige

File: frontend/src/containers/Attendance/index.jsx [hunk 1]
```diff
@@ -1,7 +1,89 @@
...
+          <MarBox />
+        
+        </Right>
+      </SubWrapper>
+
+    </Wrapper>
+  );
 }
 
 export default Attendance;
```
Comment (yell66w) [inline frontend/src/containers/Attendance/index.jsx:89]: Ano po ito? Bat nakalagay sa attendance?

Comment (yell66w) [reply frontend/src/containers/Attendance/index.jsx:89]: conflict siguro to sa #201

**Review by yell66w — CHANGES_REQUESTED** [review]
Pachange po ako nung component

**Review by yell66w — APPROVED** [review]

**Pull request merged**